### ISSUE TITLE
Uncaught TypeError: Cannot read property 'markBufferRange' of undefined

### ISSUE THREAD

**Issue by nicksergeant** [t9md/atom-narrow#275]
**Steps to reproduce:**

1. Run `narrow:search`
2. Search for `start`

I'm not entirely sure if this error is due to there being so many search results, but I was able to reproduce it on multiple unrelated projects. I was not able to reproduce it on projects with only a few test files.

**Atom**: 1.24.0 x64
**Electron**: 1.6.16
**OS**: Mac OS X 10.12.6
**Thrown From**: [narrow](https://github.com/t9md/atom-narrow) package 0.57.0

### Stack Trace

Uncaught TypeError: Cannot read property 'markBufferRange' of undefined

```
At /Users/nsergeant/.atom/packages/narrow/lib/grouped-highlight.js:29

TypeError: Cannot read property 'markBufferRange' of undefined
    at GroupedHighlight.markRange (/packages/narrow/lib/grouped-highlight.js:29:38)
    at markRange (/packages/narrow/lib/narrow-editor.js:378:26)
    at NarrowEditor.highlightItems (/packages/narrow/lib/narrow-editor.js:424:9)
    at Ui.renderItems (/packages/narrow/lib/ui.js:644:27)
    at reducers.reduce (/packages/narrow/lib/item-reducer.js:212:35)
    at Array.reduce (native)
    at ItemReducer.reduce (/packages/narrow/lib/item-reducer.js:211:19)
    at refreshDisposables.CompositeDisposable.onDidUpdateItems.items (/packages/narrow/lib/ui.js:734:26)
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:25:20)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:141:34)
    at Ui.emitDidUpdateItems (/packages/narrow/lib/ui.js:59:45)
    at Provider.updateItems (/packages/narrow/lib/provider/provider.js:245:13)
    at Search.updateItems (/packages/narrow/lib/provider/search.js:104:19)
    at updateItemsIfNotModified (/packages/narrow/lib/provider/search.js:88:12)
    at stdout (/packages/narrow/lib/searcher.js:95:28)
    at Socket.<anonymous> (/Applications/Atom.app/Contents/Resources/app/src/buffered-process.js:212:17)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    at Pipe.onread (net.js:551:20)
```

### Commands

```
     -0:06.0 core:close (input.hidden-input)
     -0:04.5.0 narrow:search (input.hidden-input)
     -0:04.4.0 vim-mode-plus:activate-insert-mode (atom-text-editor.editor.narrow.narrow-editor.search.vim-mode-plus.normal-mode.prompt)
```

### Non-Core Packages

```
busy-signal 1.4.3 
cursor-history 0.13.0 
django-templates 0.5.0 
git-blame 1.5.0 
git-time-machine 1.5.9 
hyperclick 0.1.5 
intentions 1.1.5 
jump-to-import 3.6.5 
language-haml 0.25.2 
linter 2.2.0 
linter-eslint 8.4.1 
linter-ui-default 1.6.10 
narrow 0.57.0 
persistent-undo 1.3.0 
prettier-atom 0.50.0 
sync-settings 0.8.3 
vim-mode-plus 1.28.0 
```

**Comment by t9md** [comment]
Thanks for reporting, fixed and released as v0.57.1.

**Comment by nicksergeant** [comment]
Awesome, thanks for the quick fix!
